src/site/markdown/eclipse-versions.md.vm

```diff
@@ -26,6 +26,7 @@ their compatibility with major Eclipse releases.
 
 formatter-maven-plugin | Eclipse<br/>Name | Eclipse<br/>Version | JDT Core | JDK<br/>Version
 -----------------------|------------------|---------------------|----------|----------------
+2.24                   | 2023-12          | 4.30                | 3.36     | 11
```

Comment:
Make sure this says 17 for jdk requirement.

Reply:
I just updated the branch.